.browserslistrc

```diff
@@ -1,9 +1,14 @@
 [production]
-last 2 Firefox versions
-last 2 Chrome versions
-last 2 Safari versions
-> 0.25%
-not ie 11
+Firefox > 0 and last 2 years and > 0.01%
+Chrome > 0 and last 2 years and > 0.01%
+Safari > 0 and last 2 years and > 0.01%
+Edge > 0 and last 1 year and > 0.01%
+Opera > 0 and last 2 year and > 0.01%
+> 0.2%
+not op_mini all
+not and_uc < 100
+not android < 100
+not dead
```

Comment:
Why are we expanding the browser list and why is this what we're going with?

Reply:
I added details to the PR description.